toolkit/resources/manifests/package/local.repo

```diff
@@ -5,6 +5,7 @@ enabled=1
 gpgcheck=0
 skip_if_unavailable=1
 sslverify=0
+priority=1
```

Comment:
While we're here, maybe we can also add a comment at the top explaining the reasoning behind this order of repositories, so that we remember what we were doing later on. Or for the future devs.

When I think about it now, I believe our design was:
- Start with Azure Linux's toolchain packages on purpose, even if they would overwrite the local ones, to guarantee a stable, known build environment.
- For everything else, prefer the local packages first in case someone wants to build with their version of a package we already have in PMC.
- PMC for the remaining stuff.